robyn/processpool.py

```diff
@@ -2,9 +2,14 @@
 import signal
 import sys
 import webbrowser
-from typing import Dict, List, Optional
+from typing import Dict, List, Optional, TYPE_CHECKING
 
-from multiprocess import Process
+# Workaround while multiprocess does not support mypy type checking
+# see https://github.com/uqfoundation/multiprocess/issues/128#issuecomment-2188208560
+if TYPE_CHECKING:
+    from multiprocessing import Process
+else:
+    from multiprocess import Process
```

Comment:
same as above please

Reply:
Are you happy with my response to the earlier one?

Reply:
Ok, now fixed with # type: ignore